### DISCUSSION TITLE
Is content proposal available for stream files too?

### DISCUSSION THREAD

**Discussion by DelYv** [codefori/vscode-ibmi#1160] [Q&A]
I work with stream files, and import SRVPGM's by using \include. In RDI the content proposal works fine, but in VS code, I didn't manage to use the content proposal functionality. How should I proceed?

**Comment by worksofliam** [comment]
Works fine usually.

**Comment by DelYv** [comment]
Our iASP was not supported from Code for I I presume, but it seems to be working properly since August. Thanks for your work! 
We are still encountering issues with nested includes, as described in this [post](https://github.com/orgs/halcyon-tech/discussions/1072#discussioncomment-6860957). We have attempted to use relative paths in the /includes, but have not yet been successful. We don't know if it is possible with our iASP and stream files...
